### PR TITLE
feat(cla): add cla signing and administration steps

### PR DESCRIPTION
**Pull request by lewisdaly**
- adds instructions and screenshots for how to sign the cla
- adds a cla step to the new contributor checklist

### CODE REVIEW

**Review by simeonoriko — APPROVED** [review]

File: contributors-guide/signing-the-cla.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Signing the CLA
+
+Mojaloop has a [Contributor Licence Agreement (CLA)](https://github.com/mojaloop/mojaloop/blob/master/CONTRIBUTOR_LICENSE_AGREEMENT.md) which clarifies the intelletual property for contributions from individuals or entities.
```
Comment (elnyry-sam-k) [inline contributors-guide/signing-the-cla.md:3]:
```suggestion
Mojaloop has a [Contributor License Agreement (CLA)](https://github.com/mojaloop/mojaloop/blob/master/CONTRIBUTOR_LICENSE_AGREEMENT.md) which clarifies the intellectual property rights for contributions from individuals or entities.
```

File: contributors-guide/signing-the-cla.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+To ensure every developer has signed the CLA, we use [CLA Assistant](https://cla-assistant.io/), a well maintained, open source tool which checks to make sure that a contributor has signed the CLA before allowing a pull request to be merged.
+
+## How to sign the CLA
+
+1. Open a pull request to any Mojaloop repository
```
Comment (elnyry-sam-k) [inline contributors-guide/signing-the-cla.md:9]:
```suggestion
1. Open a pull request on any Mojaloop repository
```

File: contributors-guide/signing-the-cla.md (hunk 1 above)
Comment (lewisdaly) [reply contributors-guide/signing-the-cla.md:3]: Thanks - yeah I always forget to spellcheck...

File: contributors-guide/signing-the-cla.md (hunk 2 above)
Comment (lewisdaly) [reply contributors-guide/signing-the-cla.md:9]: Haha that's more of a semantic issue I'd say - I'm not fussed either way

**Comment by elnyry-sam-k** [comment]
Thanks @lewisdaly - please review a couple of minor comments..

**Review by simeonoriko — DISMISSED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**